data/frameSimulator/src/MercuryFrameSimulatorPlugin.cpp

```diff
@@ -88,16 +128,16 @@ namespace FrameSimulator {
     void MercuryFrameSimulatorPlugin::create_frames(const int &num_frames) {
         LOG4CXX_DEBUG(logger_, "Creating Frames");
 
-        // Build array of pixel data used for each frame
-        const int num_pixels = Mercury::pixel_columns_per_sensor * Mercury::pixel_rows_per_sensor;
-        uint16_t* pixel_data = new uint16_t[num_pixels];
+        // Build array of pixel data used for each frame NOW COVERED IN READING JSON
+        // const int num_pixels_ = Mercury::pixel_columns_per_sensor * Mercury::pixel_rows_per_sensor;
```

Comment:
If commented out code is no longer needed it should be removed ideally

Reply:
Aye, thought I'd done that but it slipped through the net. Done